source/server/configuration_impl.h

```diff
@@ -130,9 +185,16 @@ class MainImpl : Logger::Loggable<Logger::Id::config>, public Main {
     std::list<NetworkFilterFactoryCb> filter_factories_;
   };
 
-  static std::list<NetworkFilterConfigFactory*>& filterConfigFactories() {
-    static std::list<NetworkFilterConfigFactory*> filter_config_factories;
-    return filter_config_factories;
+  static std::unordered_map<std::string, NetworkFilterConfigFactory*>& filterConfigFactories() {
+    static std::unordered_map<std::string, NetworkFilterConfigFactory*>* filter_config_factories =
```

Comment:
oki doki, i have not inspected all the code, but looks like https://github.com/lyft/envoy/blob/master/source/server/config/network/http_connection_manager.h#L117 has a similar issue as you pointed out @htuch